### ISSUE TITLE
Can someone explain the process of importing this to Visual Studio?

### ISSUE THREAD

**Issue by mmelatti** [peters/openalpr-windows#17]

**Comment by Co8yy** [comment]
Did this question ever get answered? is it posted some where? thanks(Newbi searching for answers)

**Comment by mmelatti** [comment]
Yeah eventually I figured out the issue. There are compatibility issues with visual studio and linker. I believe the fix is retrograding. I think retrograding my only be an option for the premium paid version that does not come cheap

**Comment by Co8yy** [comment]
I have been working on openalpr program off and on for over six months. I tried the Matt hill option, but homebrew/science formulea is no longer an option (brew install --HEAD openalpr) and github page is not really for beginners per si no fault to him. I've watched countless tutorials on youtube seeing how I am self taught/learning (nights and weekends) new coder working on a code dominated project. I find learning to code rewarding and at the same time frustrating when a person doesn't have the understanding do what may seem simple to others. I am using a mac os with virtualBox hosting win7 x64 bit. I have downloaded VS2015 with update 3 and I have VS2017 on Mac side. I have not been able to get the openalpr to function in any capacity even using Chris Dahms github or youtube tutorial. VS2015 community version gave me errors the my colleagues could not figure out. I revamped some of the code to make it process more efficiently.  Do you know of any update resources I could follow through with plate number recognition? Thanks

**Comment by Co8yy** [comment]
Have you tried your fix in any other Visual Studio versions? I just downloaded 2017 version last week. My 2015 VS was a trail version and no longer will open. Thankfully I can still get to the .cpp files.

**Comment by mmelatti** [comment]
for working with alpr, I would use your virtual box and work in Ubuntu if you can for any type of development you want to do. The original creators designed alpr for a linux box. Their proprietary webapp backend uses a linux box and many of the features of alpr will only work on a linux box: aka their video detection takes advantage of Linux Daemon services, etc....
